cl/users/views.py

```diff
@@ -712,20 +713,69 @@ def moosend_webhook(request: HttpRequest) -> HttpResponse:
 @login_required
 @never_cache
 def view_webhooks(request: AuthenticatedHttpRequest) -> HttpResponse:
+    """Render the webhooks page"""
+    return render(
+        request,
+        "profile/webhooks.html",
+        {
+            "private": True,
+            "page": "api_webhooks",
+        },
+    )
+
+
+@login_required
+@never_cache
+def view_webhooks_list(request: AuthenticatedHttpRequest) -> HttpResponse:
+    """List webhooks for the current user"""
+    webhooks = request.user.webhooks.all().order_by("date_created")
+    return render(
+        request,
+        "includes/webhooks-list.html",
+        {"webhooks": webhooks},
+    )
+
+
+@login_required
+@require_http_methods(["POST"])
+def delete_webhook(request: AuthenticatedHttpRequest, pk: int) -> HttpResponse:
+    """Handle the deletion of a webhook"""
+    webhook = get_object_or_404(Webhook, pk=pk, user=request.user)
+    webhook.delete()
+    return HttpResponse(headers={"HX-Trigger": "webhooksListChanged"})
+
+
+@login_required
+@require_http_methods(["POST", "GET"])
+def webhook_form(request: HttpRequest) -> HttpResponse:
+    """Handle the creation or editing of a webhook"""
+    pk = ""
+    if "pk" in request.GET:
+        pk = request.GET["pk"]
+    if "pk" in request.POST:
+        pk = request.POST["pk"]
```

Comment:
Maybe this comment won't matter if we redo this, but why do we allow a GET request here?